html/index.html

```diff
@@ -16,7 +16,9 @@
 working. Further configuration is required.</p>
 
 <p>For online documentation and support please refer to
-<a href="https://openresty.org/">openresty.org</a>.<br/></p>
+<a href="https://openresty.org/">openresty.org</a>.<br/>
+Commercial support is available at
```

Comment:
Better introduce a new paragraph here?

Reply:
There is no space in the default one shipped with NGINX. But if you want I can certainly add it.

Reply:
@dndx Not a big deal. We can merge the current form for now :)